src/include/distributed/pg_dist_object.h

```diff
@@ -0,0 +1,29 @@
+/*-------------------------------------------------------------------------
```

Comment:
This file should be merged into src/include/distributed/metadata/pg_dist_object.h. Two issues:
1. Both start with `#ifndef PG_DIST_OBJECT_H #define PG_DIST_OBJECT_H`,
2. Both define `Anum_pg_dist_object...`.